sign-and-combine-ecdsa/nodejs/src/index.ts

```diff
@@ -0,0 +1,186 @@
+import * as ethers from "ethers";
+import { LitContracts } from "@lit-protocol/contracts-sdk";
+import { LitNetwork, LIT_RPC, LIT_CHAINS } from "@lit-protocol/constants";
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import {
+  LitAbility,
+  LitActionResource,
+  LitPKPResource,
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+} from "@lit-protocol/auth-helpers";
+
+import { getEnv } from "./utils";
+import { litActionCode } from "./litAction";
+
+const ETHEREUM_PRIVATE_KEY = getEnv("ETHEREUM_PRIVATE_KEY");
+let capacityTokenId = getEnv("CAPACITY_TOKEN_ID");
+let LIT_PKP_PUBLIC_KEY = getEnv("LIT_PKP_PUBLIC_KEY");
+let LIT_PKP_ETH_ADDRESS = getEnv("LIT_PKP_ETH_ADDRESS");
+
+const chain = "sepolia";
+const rpcUrl = LIT_CHAINS[chain].rpcUrls[0];
+const chainId = LIT_CHAINS[chain].chainId;
+
+const ethersWallet = new ethers.Wallet(
+  ETHEREUM_PRIVATE_KEY,
+  new ethers.providers.JsonRpcProvider(rpcUrl)
+);
+
+const ethersProvider = new ethers.providers.JsonRpcProvider(rpcUrl);
+
+export const signAndCombineAndSendTx = async () => {
+  let litNodeClient: LitNodeClient;
+
+  const yellowstoneEthersWallet = new ethers.Wallet(
+    ETHEREUM_PRIVATE_KEY,
+    new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)
+  );
+  const litContracts = new LitContracts({
+    signer: yellowstoneEthersWallet,
+    network: LitNetwork.DatilTest,
+    debug: false,
+  });
+
+  await litContracts.connect();
```

Comment:
Should be within the try/catch to be consistent with other examples. Also can you add the console logs around connecting the `LitContracts` client?

```ts
console.log("🔄 Connecting LitContracts client to network...");
    const litContracts = new LitContracts({
      signer: ethersSigner,
      network: LIT_NETWORK,
    });
    await litContracts.connect();
    console.log("✅ Connected LitContracts client to network");
```